src/main/java/com/kodilla/ecommercee/domains/User.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.ecommercee.domains;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "USERS")
+public class User {
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "USER_ID")
+    private Long id;
+
+    @Getter
+    @Column(name = "USERNAME")
+    private String username;
+
+    @Getter
+    @Column(name = "STATUS")
+    private String status;
+
+    @Getter
+    @Column(name = "USER_KEY")
+    private Long userKey;
```

Comment:
Key powinien być typu String

Reply:
W JSON encji "user" nie jest typu String, więc long powinno być okej

Reply:
w Controlerze jest String, pozatym klucz traktujemy jako ciąg znaków a nie liczbę wiec powinien być String

Reply:
Zgadza się, że w Controlerze jest String, natomiast w specyfikacji projektu jest typ liczbowy, więc jest zgodnie ze specyfikacją na bazie której miała powstać encja.

Jeżeli to miałoby być zmienione na String to w specyfikacji też należałoby to zmienić.

Reply:
ok, faktycznie nie ma apostrofów w JSON, mój błąd